src/ui/layout/qgslayoutlegendwidgetbase.ui

```diff
@@ -63,9 +63,9 @@
       <property name="geometry">
        <rect>
         <x>0</x>
-        <y>0</y>
-        <width>359</width>
-        <height>2256</height>
+        <y>-2299</y>
```

Comment:
```suggestion
        <y>0</y>
```
There is a risk the widget opens, scrolled at the bottom, right? Or do I misunderstand these values?